cpp/include/kvikio/file_handle.hpp

```diff
@@ -376,8 +376,9 @@ class FileHandle {
    */
```

Comment:
Update docstring to mention gds_threshold as well, and interaction with task_size.